test/prototype_common_utils.py

```diff
@@ -34,26 +133,104 @@ def __iter__(self):
         yield self.args
         yield self.kwargs
 
-    def __str__(self):
-        def short_repr(obj, max=20):
-            repr_ = repr(obj)
-            if len(repr_) <= max:
-                return repr_
+    def load(self, device="cpu"):
+        args = tuple(arg.load(device) if isinstance(arg, TensorLoader) else arg for arg in self.args)
+        kwargs = {
+            keyword: arg.load(device) if isinstance(arg, TensorLoader) else arg for keyword, arg in self.kwargs.items()
+        }
+        return args, kwargs
 
-            return f"{repr_[:max//2]}...{repr_[-(max//2-3):]}"
+    def __repr__(self):
+        def better_repr(obj):
```

Comment:
We no longer need to shorten the `repr` here, since that was mostly done for tensors and they are now replaced with `TensorLoader`'s.